guides/source/6_0_release_notes.md

```diff
@@ -151,25 +151,44 @@ Please refer to the [Changelog][action-cable] for detailed changes.
 
 ### Removals
 
+*   Replace `ActionCable.startDebugging()` and `ActionCable.stopDebugging()`
+    with `ActionCable.logger.enabled`.
+    ([Pull Request](https://github.com/rails/rails/pull/34370))
+
 ### Deprecations
 
+There are no deprecations for Action Cable in Rails 6.0.
```

Comment:
We are not following this style elsewhere but I like it. Can we prefix it with `*` like other entries?